README.md

```diff
@@ -1,2 +1,11 @@
 # mappers-go
 KubeEdge Device Mappers written in go, it works for KubeEdge 1.4 version and later.
+
+## Construction method
+### 1 mappers
+
+
+### 2 mapper-go-sdk
+
+SDK provide RESTful and you can connect your device fastly.
```

Comment:
What's meaning of `SDK provide RESTful`? It's better to introduce how the the mapper SDK work here.